chapters/ch06.md

```diff
@@ -1,45 +1,54 @@
 
 ->
+Bạn có một ứng dụng ảnh mèo phân khúc theo bốn thị trường lớn nhất: (i) Hoa Kỳ, (ii) Trung Quốc, (iii) Ấn Độ, và (iv) Khu vực khác. Để tạo tập phát triển và tập kiểm tra, giả định như chúng ta lấy dữ liệu từ Hoa Kỳ và Ấn Độ để tạo tập phát triển; Trung Quốc và Khu vực khác để tạo tập kiểm tra. Nói cách khác, chúng ta có thể chọn ngẫu nhiên hai trong số bốn phân khúc làm tập phát triển và hai phân khúc còn lại làm tập kiểm tra.
 
 Once you define the dev and test sets, your team will be focused on improving dev set performance. Thus, the dev set should reflect the task you want to improve on the most: To do well on all four geographies, and not only two.
 
 ->
+Một khi bạn đã phân loại tập phát triển và tập kiểm tra, đội của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Bởi vậy, tập phát triển phải đại diện tốt nhất cho dữ liệu của nhiệm vụ muốn cải thiện: Đại diện không chỉ hai mà cả bốn thị trường.
 
 There is a second problem with having different dev and test set distributions: There is a chance that your team will build something that works well on the dev set, only to find that it does poorly on the test set. I’ve seen this result in much frustration and wasted effort. Avoid letting this happen to you.
 
 ->
+Vấn đề thứ hai với việc tập phát triển và tập kiểm tra có phân phối khác nhau: Có khả năng đội của bạn xây dựng được một mô hình hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã rất thất vọng và lãng phí nhiều thời gian với những mô hình như vậy. Hãy tránh để điều này xảy ra với bạn.  
 
 As an example, suppose your team develops a system that works well on the dev set but not the test set. If your dev and test sets had come from the same distribution, then you would have a very clear diagnosis of what went wrong: You have overfit the dev set. The obvious cure is to get more dev set data.
 
 ->
+Ví dụ, đội của bạn phát triển một hệ thống thực thi tốt trên tập phát triển nhưng kém trên tập kiểm tra. Nếu tập phát triển và tập kiểm tra có cùng một phân phối thì bạn có thể chuẩn đoán ngay vấn đề: Bạn đã overfit tập phát triển. Cách xử lý hiển nhiên nhất đó là bổ sung thêm dữ liệu cho tập phát triển.
 
 But if the dev and test sets come from different distributions, then your options are less clear. Several things could have gone wrong:
 
 ->
+Nhưng nếu tập phát triển và tập kiểm tra được tạo từ các phân phối khác nhau thì việc chuẩn đoán vấn đề sẽ phức tạp hơn. Các vấn đề có thể xảy ra là:   
 
 1. You had overfit to the dev set.
 
 ->
+Bạn overfit tập phát triển.
 
 2. The test set is harder than the dev set. So your algorithm might be doing as well as could be expected, and no further significant improvement is possible.
 
 ->
+Tập kiểm tra khó hơn tập phát triển. Thuật toán của bạn lúc này đã làm tốt nhất và không thể cải thiện thêm, hoặc cải thiện không đáng kể.  
```

Comment:
Mình đề xuất dịch câu thứ hai:
"Thuật toán của bạn có thể đã hoạt động tốt trên tập phát triển, và không thể có thêm cải thiện đáng kể.". Việc thêm cụm "trên tập phát triển" làm cho câu văn rõ nghĩa hơn.

Reply:
* "Tập kiểm tra khó hơn tập phát triển" -> maybe là "Dữ liệu tập kiểm tra đa dạng (phức tạp) hơn tập phát triển"